app/assets/javascripts/rights_note.js

```diff
@@ -0,0 +1,16 @@
+/* toggle rights note field based on which rights statement is selected */
+$(document).ready(function(){
+    $('.rights-statement').change(rights_note_visibility);
+});
+
+function rights_note_visibility() {
+    var stmt = $('.rights-statement');
+    var note = $('.rights-note');
+    if ( stmt.val() == 'http://rightsstatements.org/vocab/NKC/1.0/' ) {
```

Comment:
I wonder if we could attach the notable boolean to the `<option>` as an attribute or something to avoid duplicating the logic of which statements are notable here?